docs/reference/testing.md

```
@@ -396,6 +396,38 @@ describe verify
 
 ---
 
+## Test Description Style
+
+`it` descriptions should start with `should` so they read naturally as complete sentences in test output:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)